tests/backup/backup_test.go

```diff
@@ -7075,6 +7075,132 @@ var _ = Describe("{RestartBackupPodDuringBackupSharing}", func() {
 	})
 })
 
+// This testcase verifies schedule backup creation with all namespaces.
+var _ = Describe("{ScheduleBackupCreationAllNS}", func() {
+	var contexts []*scheduler.Context
+	var appContexts []*scheduler.Context
+	var backupLocationName string
+	var backupLocationUID string
+	var cloudCredUID string
+	var bkpNamespaces []string
+	var cloudAccountName string
+	var backupName string
+	var schBackupName string
+	var schPolicyUid string
+	var restoreName string
+	var clusterStatus api.ClusterInfo_StatusInfo_Status
+	var testrailID = 58015 // testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/58015
+	namespaceMapping := make(map[string]string)
+	labelSelectors := make(map[string]string)
+	cloudCredUIDMap := make(map[string]string)
+	backupLocationMap := make(map[string]string)
+	bkpNamespaces = make([]string, 0)
+	timeStamp := strconv.Itoa(int(time.Now().Unix()))
+	periodicPolicyName := fmt.Sprintf("%s-%s", "periodic", timeStamp)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("ScheduleBackupCreation", "Create schedule backup creation with all namespaces", nil, testrailID)
+		log.Info("Application installation")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+	})
+
+	It("Schedule Backup Creation with all namespaces", func() {
+		Step("Validate deployed applications", func() {
+			ValidateApplications(contexts)
+		})
+		log.Info(bkpNamespaces)
+		Step("Adding Cloud Account", func() {
+			log.InfoD("Adding cloud account")
+			providers := getProviders()
+			for _, provider := range providers {
+				cloudAccountName = fmt.Sprintf("%s-%v", provider, timeStamp)
+				cloudCredUID = uuid.New()
+				cloudCredUIDMap[cloudCredUID] = cloudAccountName
+				CreateCloudCredential(provider, cloudAccountName, cloudCredUID, orgID)
+			}
+		})
+
+		Step("Adding Backup Location", func() {
+			providers := getProviders()
```

Comment:
if there is no change in providers, declare it at testcase level and reuse ir

Reply:
Done